ied-be/src/templates/invoices/invoice.css

```
@@ -0,0 +1,285 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Inter", system-ui, -apple-system, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)